test/data/testvib.py

```diff
@@ -67,6 +67,16 @@ def testvibsyms(self):
         """Is the length of vibsyms correct?"""
         self.assertEqual(len(self.data.vibsyms), self.numvib)
 
+    entropy_places = 3
+
+    def testtemperature(self):
+        """Is the temperature 298.15 K?"""
+        self.assertAlmostEqual(298.15, self.data.temperature)
+
+    def testentropy(self):
+         """Is the entropy reasonable"""
+         self.assertAlmostEqual(0.0001462623335480945, self.data.entropy, self.entropy_places)

```

Comment:
This looks like a test we could move to GenericIRTest, and just customize the value in the package-specific test classes.

Reply:
Which tests do you think we could move to GenericIRTest? We could move tests for temperature and pressure as well, for instance.

Reply:
I was thinking only about the entropy at this time, since that is what you are changing.